### PR TITLE
moved AuthProvider to the component, axios to the services

### PR DESCRIPTION
**Pull request by halyna-k**

### CODE REVIEW

File: src/components/AuthProvider/AuthProvider.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Auth0Provider } from "@auth0/auth0-react";
+
+const AUTH0_USER = "dev-xsron5cxoe45bn57.us.auth0.com"
+const AUTH0_KEY = "zmgQ6uO27tIDiDuJjGC4ZF0M3l0OySlv"
+// const AUTH0_USER = process.env.DOMAIN
+// const AUTH0_KEY = process.env.CLIENT_ID
+// const AUTH0_SECRET = process.env.CLIENT_SECRET
```
Comment (dimalian) [inline src/components/AuthProvider/AuthProvider.js:7]: Save the auth0 data in the .env file and share this file throw Slack with the dev team

**Comment by dimalian** [comment]
You forked the project and are making changes from your repository. This is not correct, you need to clone the project and make changes directly in the current repository. Ask @djek195 how he fixed this.

File: src/components/AuthProvider/AuthProvider.js (hunk 1 above)
Comment (halyna-k) [reply src/components/AuthProvider/AuthProvider.js:7]: > 

Okay

**Comment by halyna-k** [comment]
> You forked the project and are making changes from your repository. This is not correct, you need to clone the project and make changes directly in the current repository. Ask @djek195 how he fixed this.

Thanks a lot. Already done.

**Pull request closed without merging**